test/TestApps/ToastNotificationsDemoApp/main.cpp

```diff
@@ -113,15 +113,17 @@ std::wstring GetEnumString(winrt::AppNotificationSetting const& setting)
     return enumMapping[setting];
 }
 
-winrt::AppNotification CreateToastNotification(winrt::hstring message)
-{
-    winrt::hstring xmlPayload{ L"<toast>" + message + L"</toast>" };
-    return winrt::AppNotification(xmlPayload);
-}
-
 winrt::AppNotification CreateToastNotification()
 {
-    return CreateToastNotification(L"intrepidToast");
```

Comment:
Shell was having a pretty hard time handling this toast. We need to figure out what went wrong with the schema validation, I am concerned this could an issue on the Toast History.